Buildings/Experimental/OpenBuildingControl/CDL/SetPoints/HotWaterTemperatureReset.mo

```diff
@@ -0,0 +1,157 @@
+within Buildings.Experimental.OpenBuildingControl.CDL.SetPoints;
+block HotWaterTemperatureReset
+  "Block to compute the supply and return set point of heating systems"
+
+  parameter Real m = 1.3 "Exponent for heat transfer";
+  parameter Modelica.SIunits.Temperature TSup_nominal "Supply temperature"
+    annotation (Dialog(group="Nominal conditions"));
+  parameter Modelica.SIunits.Temperature TRet_nominal "Return temperature"
+    annotation (Dialog(group="Nominal conditions"));
+  parameter Modelica.SIunits.Temperature TRoo_nominal = 293.15
+    "Room temperature"
+    annotation (Dialog(group="Nominal conditions"));
+  parameter Modelica.SIunits.Temperature TOut_nominal "Outside temperature"
+    annotation (Dialog(group="Nominal conditions"));
+
+  parameter Boolean use_TRoo_in = false
+    "Get the room temperature set point from the input connector"
+    annotation(Evaluate=true, HideResult=true);
+  parameter Modelica.SIunits.Temperature TRoo = 293.15
+    "Fixed value of room temperature set point"
+    annotation(Dialog(enable = not use_TRoo_in));
+  parameter Modelica.SIunits.TemperatureDifference dTOutHeaBal(displayUnit="K") = 8
+    "Offset for heating curve";
+  Interfaces.RealInput TRoo_in(final quantity="ThermodynamicTemperature",
+                                               final unit = "K", displayUnit = "degC", min=0) if
+          use_TRoo_in "Room air temperature set point"
+    annotation (Placement(transformation(extent={{-139,-80},{-99,-40}})));
```

Comment:
Remove `use_TRoo_in` (and `TRoo_in_internal` and the parameter `TRoo`. 
For CDL, we do not want to allow conditionally removed connectors as I don't anticipate that control system will be able to support such a feature.

For consistency with guideline 36, we should use zone instead of room, and rename `Roo` to `Zon`. In this case, please rename the input `TRoo_in` to `TSetZon` (then, it uses the same name as `ASHRAE.G36.VAVSingleZoneTSupSet`)